.github/ISSUE_TEMPLATE/bug-en.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug
+about: Create a bug report to help us improve
+title: "[BUG] Feature name - Short description of the bug"
+labels: ''
+assignees: ''
+
+---
+
+## Required information
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+If you found another bug report related to this issue on GitHub, please mention its number e.g. #7890
+
+**To Reproduce**
```

Comment:
```suggestion
**To Reproduce**
<!-- Mandatory -->
```

Reply:
Having it in the preceding section header is sufficient.